packages/cli/src/util/client.ts

```diff
@@ -151,7 +152,9 @@ export default class Client extends EventEmitter {
     const result = await reauthenticate(this, error);
 
     if (typeof result === 'number') {
-      this.output.prettyError(error);
+      if (error instanceof APIError) {
+        this.output.prettyError(error);
+      }
```

Comment:
Do we need to print a generic error message if its not `APIError`? Otherwise it will exit without telling the user anything?

Reply:
Good idea.